src/ui/public/agg_types/buckets/terms.js

```diff
@@ -92,9 +93,14 @@ export default function TermsAggDefinition(Private) {
           $scope.$watch('responseValueAggs', updateOrderAgg);
           $scope.$watch('agg.params.orderBy', updateOrderAgg);
 
+          $scope.rejectAgg = function (agg) {
+            // aggFilter elements all starts with a '!'
+            // so the index of agg.type.name in a filter is 1 if it is included
+            return Boolean(aggFilter.find((filter) => filter.indexOf(agg.type.name) === 1));
+          };
```

Comment:
I know it looks bad, but I cannot use `_.contains` because it uses `SameValueZero` for equality comparisons: https://github.com/lodash/lodash/blob/3.10.1/doc/README.md#_includescollection-target-fromindex0

I just want to reuse the same array as the one used for the schema. However, I cannot reuse the [prop_filter](https://github.com/elastic/kibana/blob/cd760781b202e74c0dc83edcd8e36341043a42f7/src/ui/public/filters/_prop_filter.js) method because I want to have a function that works on a single element, not an array. That is because of https://github.com/elastic/kibana/pull/7302/files/e28a18d6c8a08037be159631fc3f554d057bf254#diff-b171a84786952a20d0db9267b53156c0R12 essentially. Does it make sense ?